com.unity.netcode.gameobjects/Components/ServerNetworkAnimator.cs

```diff
@@ -0,0 +1,13 @@
+using UnityEngine;
+
+namespace Unity.Netcode.Components
+{
+    /// <summary>
+    /// A prototype component for syncing Mecanim Animator state in a server-driven manner
+    /// </summary>
+    [AddComponentMenu("Netcode/" + nameof(ServerNetworkAnimator))]
+    public class ServerNetworkAnimator : NetworkAnimator
```

Comment:
any reason why we still have serverNetworkAnimator after our discussion? Why not set the default value IsAuthorityOverAnimator in NetworkAnimator? Netcode is server authoritative, default should be that too, no?

Reply:
Our conversation yesterday (the one regarding NetworkTransform), ended up on the base-class (NetworkTransform) and two implementations (Server and ClientNetworkTransform), did it not?

 In this implementation NetworkAnimator is abstract, though strictly speaking it is not necessary and what you're proposing can be done. What would be the practical difference though?

Reply:
Our discussion yesterday _is_ the reason for ServerNetworkAnimator-> NetworkAnimator existence :P